webapp/templates/jury/rejudging.html.twig

```diff
@@ -109,6 +109,42 @@
         {% include 'jury/partials/rejudging_matrix.html.twig' %}
     </div>
 
+    {% if stats %}
+        <h2>Statistics over repeated rejudgings</h2>
+        <h4>Runtime spread</h4>
+        <ul>
+        {% for spread in stats.runtime_spread %}
+            <li>{{ spread }}</li>
+        {% endfor %}
+        </ul>
+        {% if stats.judging_runs_differ %}
+            <h4>Judging Run Differences</h4>
+            <ul>
+                {% for diff in stats.judging_runs_differ %}
+                    <li>s{{ diff }}</li>
+                {% endfor %}
+            </ul>
+            {% if stats.judging_runs_differ_overflow > 0 %}
+                <em>Left out {{ stats.judging_runs_differ_overflow }} submissions with different judging results.</em>
+            {% endif %}
+        {% endif %}
+        {% if stats.judgehost_stats | length > 1 %}
+            <h4>Judgehost stats</h4>
+            <table>
```

Comment:
This might look neater with some kind of border, perhaps add:
`class="table table-sm"` and/or `table-striped` and/or `table-border`

Reply:
Done.